browser/ui/brave_browser.cc

```diff
@@ -85,8 +86,13 @@ void BraveBrowser::ScheduleUIUpdate(content::WebContents* source,
   // We need to update sidebar UI only when current active tab state is changed.
   if (changed_flags & content::INVALIDATE_TYPE_URL) {
     if (source == tab_strip_model_->GetActiveWebContents()) {
+      // sidebar() can return a nullptr in unit tests.
```

Comment:
this doesn't seem like the right check, it should not be null unless `!sidebar::CanUseSidebar` so I think we're probably missing a test here and maybe this can crash in release cc @petemill

Reply:
So, it seems what happens here is that the sidebar gets created in `BraveBrowserView`, but `BraveBrowserView` doesn't get created in unit tests because unit tests use `TestBrowserWindow` instead of `BrowserWindow::CreateBrowserWindow` where we normally create `BraveBrowserView`. cc: @sangwoo108